### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is not descriptive for the "Average Rating" link

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#53]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Average rating" link available in the "Python" Blue window below the page header.
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is not descriptive
**_Narrator Behaviour:_**
The Narrator announces as "81 link"

**Expected Result:**
The Name property should be descriptive so that the users able to understand what is the purpose of the link
**_Suggested Fix:_**
Either "Name" property should be descriptive or "Help Text" property should be added

**User Impact:**
The visually impaired users will not be able to understand the exact purpose of the link

**Screenshot:**
![name property is not descriptive for average rating link](https://cloud.githubusercontent.com/assets/24825441/21758674/054a1870-d664-11e6-91ce-9a662fc0f694.png)

**Comment by waderyan** [comment]
Marketplace
